### PR TITLE
fence_ovm: new fence agent for Oracle VM

### PR DESCRIPTION
**Pull request by kaneter**
Hello.

Adds a fencing agent for virtual machines managed by Oracle VM.

fence-ovm uses Oracle VM Manager REST API.
https://docs.oracle.com/en/virtualization/oracle-vm/3.4/developer/vmapi-rest.html

Tested with Oracle VM Manager 3.4 on Oracle Linux Server 7.9.

### CODE REVIEW

**Comment by oalbrigt** [comment]
Thanks.

File: agents/ovm/fence_ovm.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+		message = result.get("message") if type(result) is dict else None
+		if message is None:
+			logging.error("Failed: Remote returned %d for '%s' request to %s", response.status_code, method, uri)
+		else:
+			logging.error("Failed: %s: %d: %s", method, response.status_code, result["message"])
+		fail(EC_GENERIC_ERROR)
+	return result
+
+def define_new_opts():
+	all_opt["ssl_client_cert_file"] = {
```
Comment (oalbrigt) [inline agents/ovm/fence_ovm.py:130]: You should update this to "ssl_client_certificate_file", as it's used for stdin parsing of parameters (e.g. from Pacemaker), so using the same name here will make it easier for users to create the resource based on the parameters they used in CLI testing.

Comment (kaneter) [reply agents/ovm/fence_ovm.py:130]: I  understand. After modification, I will create a new PR.

Comment (fabbione) [reply agents/ovm/fence_ovm.py:130]: You can just update this PR. no need to create a new PR each time.

Comment (kaneter) [reply agents/ovm/fence_ovm.py:130]: The modification is done.
Could you review this update in this PR ?
I'm concerned that this update is not reflected in this PR.

Comment (oalbrigt) [reply agents/ovm/fence_ovm.py:130]: Since this was already merged you'll have to create a new PR. Fabio probably didnt notice it was merged.

**Pull request merged**